test/testutils.jl

```diff
@@ -1,23 +1,44 @@
-allapprox(coords₁::C, coords₂::C; kwargs...) where {C<:CRS} =
-  all(ntuple(i -> isapprox(getfield(coords₁, i), getfield(coords₂, i); kwargs...), nfields(coords₁)))
-
-function allapprox(coords₁::C, coords₂::C; kwargs...) where {C<:Cartesian2D}
-  isapprox(coords₁.x, coords₂.x, kwargs...) && isapprox(coords₁.y, coords₂.y, kwargs...)
+svec(coords::Cartesian) = SVector(getfield(coords, :coords))
+svec(coords::CoordRefSystems.Projected) = SVector(coords.x, coords.y)
```

Comment:
We can `using CoordRefSystems: Projected` in `test/runtests.jl` to avoid the prefix in implementations.